src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/ErrorNotificationHandler.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.tasks;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.azure.servicebus.ExceptionPhase;
+import com.microsoft.azure.servicebus.IMessage;
+import com.microsoft.azure.servicebus.IMessageHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.reform.bulkscanprocessor.exceptions.InvalidMessageException;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg.ErrorMsg;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.ErrorNotificationService;
+
+import java.io.IOException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+
+public class ErrorNotificationHandler implements IMessageHandler {
+
+    private static final Logger log = LoggerFactory.getLogger(ErrorNotificationHandler.class);
+
+    private final ErrorNotificationService service;
+
+    private final ObjectMapper mapper;
+
+    static final Executor SIMPLE_EXEC = Runnable::run;
```

Comment:
are the scope of these not private ?

Reply:
It is private but was not sure how this gets registered, iirc those statics are purposefully reused across multiple instances within same server which is one of the main points on how to use futures in java. After stating the obvious - not use executors from common pull

But I must be overthinking too much. Do you think it's worth setting them private?

Reply:
yes let's keep those private to make sure we follow encapsulation